src/main/java/org/jivesoftware/openfire/plugin/passwordreset/PasswordResetTokenManager.java

```diff
@@ -154,10 +162,34 @@ public List<ResetRequest> getResetRequests() {
         }
     }
 
-    @Data
     public static class ResetRequest {
         public final String userId;
         public final String sourceAddress;
         public final Date expires;
+

```

Comment:
You're missing the toString(), hashCode() and equals() methods here. I can't remember how this is used, since you probably don't need hashCode/equals, but toString() would be useful when this is logged.